core/spec.md

```diff
@@ -5528,6 +5549,30 @@ as defined below:
   - MUST NOT be present if the Resource's `hasdocument` model attribute is
     set to `false.
 
+##### `ancestor` Attribute
+- Type: String
+- Description: A string value that points to another version based on the
+  `versionid`-attribute.
+
+  When creating a version without explicitly setting the `ancestor`
+  attribute, the server MUST set the `ancestor` to the `versionid` of the
```

Comment:
The risk here is that if later the user changes the value of the `versionid` attribute to a new value, we'll have to enforce implementations to update the `ancestor` attribute of all versions that point to the changed `versionid` to ensure accidentally creating dangling versions that don't point to a valid version anymore.